packages/devtools/lib/src/service_extensions.dart

```diff
@@ -118,19 +159,18 @@ const slowAnimations = ToggleableServiceExtensionDescription<num>._(
   gaItem: ga.slowAnimation,
 );
 
-const togglePlatformMode = ToggleableServiceExtensionDescription<String>._(
+final togglePlatformMode = ServiceExtensionDescription<String>(
   extension: 'ext.flutter.platformOverride',
-  description: 'iOS',
+  description: 'Override target platform',
   icon: FlutterIcons.phone,
-  enabledValue: 'iOS',
-  disabledValue: 'android',
-  enabledTooltip: 'Toggle iOS Platform',
-  disabledTooltip: 'Toggle iOS Platform',
+  values: ['iOS', 'android', 'fuchsia'],
```

Comment:
we should also have a value that is 
"unset" or similar
That is the default state where the value actually matches your platform.

Reply:
The starting value is currently set to whatever the platform actually is.

 I think "Platform: none" or "Platform: unset" could be kind of confusing. Even more confusing would be a dropdown whose starting value is just "Unset". I think having the starting value be consistent with the device is actually more intuitive because it helps connect the dots between the device and what the dropdown selection changes.